arci/src/traits/move_base.rs

```diff
@@ -32,12 +32,16 @@ impl BaseVelocity {
 impl std::ops::Mul<f64> for BaseVelocity {
     type Output = Self;
 
-    fn mul(self, rhs: f64) -> Self::Output {
```

Comment:
`Self{ }` is generate new instance.
But this function need self. I think it is waste action.